packages/ra-no-code/src/builders/CreateBuilder.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {
+    CreateContextProvider,
+    CreateControllerProps,
+    useCreateContext,
+    useCreateController,
+} from 'ra-core';
+import { CreateProps, CreateView, SimpleForm } from 'ra-ui-materialui';
+import { getInputFromFieldDefinition } from './getInputFromFieldDefinition';
+import { useResourceConfiguration } from '../ResourceConfiguration';
+
+export const CreateBuilder = (props: CreateProps) => {
```

Comment:
I still don't like the name 'builder'. Why not call it just `<Create>`?

Reply:
To avoid confusion with the react-admin components